modules/profile/manifests/iib/example/brokers.pp

```diff
@@ -0,0 +1,9 @@
+class profile::iib::example::nodes(
```

Comment:
Maybe it is good to make this into broker to and update the hiera data accordingly.